### DISCUSSION TITLE
M.I.B. compatability question...

### DISCUSSION THREAD

**Discussion by nazaar33** [Mr-MIBonk/M.I.B._More-Incredible-Bash#224] [General]
Hello all. I have a couple of questions about M.I.B. tools and using them on my 2021 Audi S7. I believe my MMI software is an MIB3+ software and from everything that I am able to find, the tools available via M.I.B. and other VAG applications (OBDEleven, Ross-Tech etc..) all seem to have instructions/directions on using them with systems that are all prior to MIB3+.  I am attempting to change my VIM settings in my 2021 Audi S7 to enable Video playback to a high speed setting. 

My question to you all here is: 
Can I use M.I.B. tools (on MIB3+ MMI system) to change the VIM playback speed setting to a higher speed, and if so, how? 

How can I determine if my MMI version is compatible with M.I.B.?  I looked at the compatability table for FW versions tested with M.I.B., and do not see my MMI software version listed there. My MMI Software version contains "MLB-AS_2025...." in it and the Media driver has "MIB 3P_AU_ER_..." in it, neither of which I can find on the M.I.B. site here. 
I do see see in the notes early on where it says "M.I.B. is running on Harman MHI2(Q) units (MIB 2.x) only.". That may in fact by my answer to the above questions, but I wanted to make sure by asking you all. And if M.I.B. turns out not to be compatible, does anyone know of how I can enable VIM at a higher speed?

Any help anyone can give me is very much appreciate as I have been trying to figure this out for some time now.

Thanks in advance for any help.

Mike

**Comment by harman-f** [comment]
MHI2(Q) units only ;)
